ios/PurchasesHybridCommon/PurchasesHybridCommonTests/PurchaserInfoHybridAdditionsTests.swift

```diff
@@ -78,7 +80,7 @@ class PartialMockPurchaserInfo: Purchases.PurchaserInfo {
     }
     
     override var firstSeen: Date {
-        return Date()
+        return _firstSeen
```

Comment:
having these return on-the-fly would make it impossible to compare the generated dictionaries of them with `==`, since each time `dictionary` gets called it has a new value